vars/malwareExecuteScan.groovy

```diff
@@ -0,0 +1,42 @@
+import groovy.transform.Field
+
+import com.sap.piper.GenerateDocumentation
+
+@Field String STEP_NAME = getClass().getName()
+
+@Field STEP_CONFIG_KEYS = [
+
+
+    /**
+      * Malware scanning host.
+      */
+    'host',
+    /**
+      * The file which is scanned for malware
+      */
+    'file',
+    /**
+      * The credentials used for accessing the malware scanning host
+      */
+    'credentialsId'
+
+]
+
+@Field GENERAL_CONFIG_KEYS = STEP_CONFIG_KEYS
+
+@Field PARAMETER_KEYS = GENERAL_CONFIG_KEYS
+
+@GenerateDocumentation
```

Comment:
this is not needed since docs are generated on `golang` side

Reply:
@OliverNocon : aaahhh. Can you explain how I can check the generated docu, most preferable on the level of the PR?

Reply:
@OliverNocon : how can we deal with the credentialsId? This is not available in the go metadata - there we have username, password. Should we add the credentialsId there also? Not sure how that works since that might depend on the framework we are using (of course we have some credentialsId also in the metadata as some kind of best guess ...)

Reply:
> Can you explain how I can check the generated docu, 

found it myself: `go run pkg/generator/step-metadata.go --docuDir=documentation/docs/steps/ --docuGen=true`